fix "Get app state" dubug menu in Android

TETRA2000 opened the pull request:
fixes #11

frantic commented on js/actions/test.js:
```diff
@@ -54,10 +55,21 @@ function testResetNuxes(): Action {
-      subject: `App v${version} state`,
-      message: JSON.stringify(getState(), undefined, 2),
-    }, () => {}, () => {});
+    const subjcet = `App v${version} state`;
```
did you mean `subject`?

frantic commented on js/actions/test.js:
```diff
@@ -54,10 +55,21 @@ function testResetNuxes(): Action {
+        message: message,
+      }, () => {}, () => {});
+    } else {
+      const SendIntentAndroid = require('react-native-send-intent');
```
nit: can you move this to the top of the file after other `require`s?

frantic commented:
Thank you for the fix! Please address the inline comments and I'll pull this in

TETRA2000 commented:
@frantic Thank you for your review!

I fixed my typo and changed to require on the top of file.
I conditionally require the module.
Because `react-native-send-intent`'s `index.js` uses const from java file.

diff after your review
```diff
diff --git a/js/actions/test.js b/js/actions/test.js
index 0fec6f5..c492133 100644
--- a/js/actions/test.js
+++ b/js/actions/test.js
@@ -28,6 +28,7 @@ const Parse = require('parse/react-native');
 const ActionSheetIOS = require('ActionSheetIOS');
 const Platform = require('Platform');
 const {version} = require('../env');
+const SendIntentAndroid = Platform.OS === 'android' ? require('react-native-send-intent') : undefined;

 import type { Action, ThunkAction } from './types';

@@ -55,17 +56,16 @@ function testResetNuxes(): Action {

 function testExportAppState(): ThunkAction {
   return (dispatch, getState) => {
-    const subjcet = `App v${version} state`;
+    const subject = `App v${version} state`;
     const message = JSON.stringify(getState(), undefined, 2);
     if (Platform.OS === 'ios') {
       ActionSheetIOS.showShareActionSheetWithOptions({
-        subject: subjcet,
+        subject: subject,
         message: message,
       }, () => {}, () => {});
     } else {
-      const SendIntentAndroid = require('react-native-send-intent');
       SendIntentAndroid.sendText({
-        title: subjcet,
+        title: subject,
         text: message,
         type: SendIntentAndroid.TEXT_PLAIN
       });
```

frantic commented on js/actions/test.js:
```diff
@@ -26,7 +26,9 @@
 const ActionSheetIOS = require('ActionSheetIOS');
+const Platform = require('Platform');
 const {version} = require('../env');
+const SendIntentAndroid = Platform.OS === 'android' ? require('react-native-send-intent') : undefined;
```
FYI the `require`s are resolved at build time, i.e. it doesn't matter if you do `Platform.OS === 'android'` – these will get included into the bundle anyways. Looking at [that module's source](https://github.com/lucasferreira/react-native-send-intent/blob/master/index.js) it seems it's totally safe to require this module on iOS anyways, as long as we are not calling methods on it.

frantic replied:
I take that back, sorry :) Filed https://github.com/lucasferreira/react-native-send-intent/issues/19. This is good to go, I'll merge it tomorrow. Thank you!

TETRA2000 commented:
@frantic Thank you for the review and reporting the library's issue!!

frantic commented:
Pulled in manually https://github.com/fbsamples/f8app/commit/d2f80518cbaf8a130332bcbecc0b786f7076de8f. Thanks @TETRA2000!

lucasferreira commented:
Hi folks, I've updated the *react-native-send-intent* module, now I think that are safe to import them in iOS platform.
